CMakeLists.txt

```diff
@@ -164,3 +188,9 @@ if(ENABLE_TESTING)
         )
     endif(UNIX)
 endif()
+
+# Make scripts and data files needed for testing available in an
+# out-of-source build.
```

Comment:
Comment says `scripts and data files needed` but it looks like just the scripts directory from the function call.